app/models/menu_items_menu.rb

```diff
@@ -4,7 +4,6 @@ class MenuItemsMenu < ActiveRecord::Base
 
   validates :daily_stock, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
   validates :menu_item, presence: true
-  validates :menu, presence: true
```

Comment:
This was the source of the bug we saw in the AA 'new menu' form.  That form contains a form for the menu itself, as well as a nested form for the menu_items_menus association.  The form create action was failing due to this validation failing.  It seems that AA is creating the associated item (menu_items_menus) before the primary (menu) item.  As a result, this validation fails because the menu_id in that object is nil.

We removed this validation as there seemed to be no situation where we would want to create an association record in the absence of an existing menu.  If there is such a case, we will have to revisit this.